### PR TITLE
REST API that accepts the name of a top-level region and returns its sequence checksum.

### PR DESCRIPTION
**Pull request by Mehrnaz-Charkhchi**
Add an endpoint to the metadata REST API that accepts the name of a top-level region and returns its sequence checksum.
Jira: [ENSWBSITES-2403](https://www.ebi.ac.uk/panda/jira/browse/ENSWBSITES-2403)

### CODE REVIEW

File: app/api/models/checksums.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+from pydantic import BaseModel, Field
+
+
+class Checksum(BaseModel):
+    md5: str = Field(max_length=32)
```
Comment (azangru) [inline app/api/models/checksums.py:5]: Does it also have a min length?

**Review by kamaldodiya — COMMENTED** [review]
Can you please add an example as to how request/response would like in PR description ? 
Thanks

**Comment by Mehrnaz-Charkhchi** [comment]
> Can you please add an example as to how request/response would like in PR description ? Thanks

We have that in OpenAPI spec.

File: app/api/models/checksums.py (hunk 1 above)
Comment (Mehrnaz-Charkhchi) [reply app/api/models/checksums.py:5]: What could the min length be?

**Comment by kamaldodiya** [comment]
> > Can you please add an example as to how request/response would like in PR description ? Thanks
> 
> @kamaldodiya We have that in OpenAPI spec.

Yes, I know but I want to check the response from the endpoint if it conforms to the API Spec  :)

File: app/api/models/checksums.py (hunk 1 above)
Comment (azangru) [reply app/api/models/checksums.py:5]: I was wondering, if it's an md5 checksum, doesn't it mean that it is always 32 characters?

**Comment by Mehrnaz-Charkhchi** [comment]
> > > Can you please add an example as to how request/response would like in PR description ? Thanks
> > 
> > 
> > @kamaldodiya We have that in OpenAPI spec.
> 
> Yes, I know but I want to check the response from the endpoint if it conforms to the API Spec :)

Not sure how it can help since we follow the spec in the first place, here you go:
http://0.0.0.0:8014/api/metadata/genome/a7335667-93e7-11ec-a39d-005056b38ce3/checksum/13
17dab79b963ccd8e7377cef59a54fe1c

File: app/api/models/checksums.py (hunk 1 above)
Comment (Mehrnaz-Charkhchi) [reply app/api/models/checksums.py]: I added min_length=32 so it's going to be strictly 32 characters now.

**Comment by kamaldodiya** [comment]
> > > > Can you please add an example as to how request/response would like in PR description ? Thanks
> > > 
> > > 
> > > @kamaldodiya We have that in OpenAPI spec.
> > 
> > 
> > Yes, I know but I want to check the response from the endpoint if it conforms to the API Spec :)
> 
> Not sure how it can help since we follow the spec in the first place, here you go: http://0.0.0.0:8014/api/metadata/genome/a7335667-93e7-11ec-a39d-005056b38ce3/checksum/13 17dab79b963ccd8e7377cef59a54fe1c

Well API Spec this is what we want we would like to have.
The code when in PR says we have implemented/delivered what was expected in API Spec.
Particularly for PR reviewer I would generally check 1) Did it conforms to the expectation ( in this case API spec) 2) code :) 
Now, it looks good to me.

**Review by kamaldodiya — APPROVED** [review]

**Pull request merged**